### PR TITLE
Connection banner: update based on feedback.

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

update wording of the main connection banner based on feedback on p1HpG7-5J1-p2 

Here is the new look:

screenshot 2018-10-30 at 16 57 45

#### Testing instructions:

* Fire up this PR.
* Check the full screen connection banner appearing when you first activate Jetpack; you will want to check the 2 first sections.

#### Proposed changelog entry for your changes:

* None

### CODE REVIEW

**Comment by rickybanister** [comment]
Looks good to me.

cc @Automattic/editorial for a gut check

**Review by kraftbj — DISMISSED** [review]

File: class.jetpack-connection-banner.php [hunk 1]
```diff
@@ -873,7 +873,7 @@ class="jetpack-logo"
 					?></p>
 					<p><?php
 						esc_html_e(
-							'Optimize: Upload images and videos directly to our powerful servers that present your media with lightning speed.',
+							"Optimize: Jetpack's site accelerator uploads images, videos, and static files like CSS to our powerful servers and delivers them at lightning-fast speeds to your visitors. You'll also save precious bandwidth.",
```
Comment (keoshi) [inline class.jetpack-connection-banner.php]: Minor nitpick, but we could still use the single quote marks if we keep the correct apostrophe character `’` (vs `'`). I know this is semi-urgent, so feel free to ignore this feedback.

**Review by keoshi — DISMISSED** [review]
Left a note on a minor detail, but LGTM and is ready to ship.

File: class.jetpack-connection-banner.php (hunk 1 above)
Comment (jeherve) [reply class.jetpack-connection-banner.php]: How do you actually do that `’`? :)

More seriously, is that a valid English character? I had it on my French keyboard back in the day, but it is not available on my English keyboard now.

Comment (keoshi) [reply class.jetpack-connection-banner.php]: In this specific case, you could copy it from https://github.com/Automattic/jetpack/pull/10490/files#diff-511f6341fbb24bcadaae3ada1e3c3039L917 but I'm sure it's different for multiple combinations of keyboard layouts and configurations. And it should be a valid character.

Comment (jeherve) [reply class.jetpack-connection-banner.php]: Done in 960aace3697179dacc10eb79c8daeb1c34f3d35b

**Review by kraftbj — APPROVED** [review]

**Comment by benhuberman** [comment]
Thanks for the ping, @rickybanister -- I'll review by EOD Wednesday.

**Comment by rickybanister** [comment]
Fair point @benhuberman. We will be adding even more stuff to this for next month so that would be a great time to trim the fat!

**Pull request merged**